Makefile

```diff
@@ -24,43 +22,25 @@ endif
 
 ## Functions
 get_github_latest_version = $$(docker run --rm easy_infra:latest /bin/bash -c "curl -s https://api.github.com/repos/$(1)/releases/latest | jq -r '.tag_name'")
-update_dockerfile_package = ./update.sh --package=$(1) --version=$(2)
-update_dockerfile_repo    = ./update.sh --repo=$(1) --version=$(2)
+update_dockerfile_package = ./update_components.sh --package=$(1) --version=$(2)
+update_dockerfile_repo    = ./update_components.sh --repo=$(1) --version=$(2)
 
 
 ## Rules
 .PHONY: all
 all: update build
 
 .PHONY: update
-update: update-functions update-deps
+update: update-deps update-functions
 
 .PHONY: update-deps
-update-deps: update-apt update-awscli update-github update-terraform update-yarn
+update-deps: update-apt update-requirements update-awscli update-github update-terraform update-yarn
 
 
 .PHONY: update-functions
 update-functions:
```

Comment:
You may at some point want to have python3 be a variable that defaults to python3, that the user can set in the environment

Reply:
Are you referring to the docker image or the bash command string?

Reply:
any place you reference `python3` outside of docker ( in the users env )

Reply:
It is never referenced outside of docker.